server/routes.js

```diff
@@ -71,6 +86,9 @@ exports.upstreamQuintypeRoutes = function upstreamQuintypeRoutes(
       .catch(() => res.status(503).send({ error: { message: "Config not loaded" } }));
   });
 
+  // Mention the routes which don't want to override the s-maxage value
+  const excludeRoutes = ["/qlitics.js", "/api/v1/breaking-news"];
```

Comment:
can you make this configurable, as publishers may want to not cache some other APIs, they should do it from the app level?